algolia_settings.json

```diff
@@ -60,7 +60,7 @@
    "linode-documentation-sorted"
   ],
   "searchableAttributes": [
-   "title",
+   "unordered(title)",
```

Comment:
Setting the title search to unordered means that Algolia will not care whether the search term appears earlier in the title string:
https://www.algolia.com/doc/api-reference/api-parameters/searchableAttributes/

This matters because we haven't always had a strong convention for titles. For example:
- We may have a had a guide in the past that was titled "Apache on Debian"
- The new version of this guide might be titled "How to Install Apache Web Server on Debian"
- If the title search was ordered, then it would be possible for the earlier title to be prioritized in search rankings, because `Apache` appears at the start of that title.